app/helpers/maintenance_tasks/tasks_helper.rb

```diff
@@ -70,7 +70,8 @@ def status_tag(status)
     #
     # @return [String] the description of the time running attribute.
     def time_running_in_words(run)
-      distance_of_time_in_words(0, run.time_running, include_seconds: true)
+      time_running = run.started_at + run.time_running.seconds
+      distance_of_time_in_words(run.started_at, time_running, include_seconds: true)
```

Comment:
Ok so we're trying to get a real time here, we don't even need `run.started_at`, we could use `Time.now` as the start, but we're not seeing any issues with the way this is done with `Time.at(0)`.
I assume something must be happening with activesupport, which version are you using?

Reply:
Hey sorry for the late response, the version we're using is `6.1.4.6` for both actionview and activesupport.